letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -264,7 +264,18 @@ UNLIKELY_EOF
       # Report error. (Otherwise, be quiet.)
       echo "Had a problem while installing Python packages."
       if [ "$VERBOSE" != 1 ]; then
+        echo
+        echo "pip prints the following errors: "
+        echo "====================================================="
         echo "$PIP_OUT"
+        echo "====================================================="
+        echo
+        echo "Certbot has problem setting up the virtual environment."
```

Comment:
Which string 😝? `internal compiler error: Killed (program cc1)` could happen because of other reasons. I don't feel particularly good to make unreliable guesses.

I can `grep` that if you think that is the right string to grep.